com.unity.cinemachine/Runtime/Behaviours/CinemachineBrain.cs

```diff
@@ -674,5 +711,71 @@ void PushStateToUnityCamera(ref CameraState state)
             // Send the camera updated event
             CinemachineCore.CameraUpdatedEvent.Invoke(this);
         }
+
+#if UNITY_EDITOR
+        class DebugText : IDisposable
```

Comment:
I'm feeling like the brain is having to do too much work.  I think I'd prefer these details to be hidden inside CinemachineDebug.  Can you think of a way to do that?

I'm imagining CinemachineDebug to be a service that assigns text widgets to any clients (e.g. CmBrain) who register and request a place to print on-screen.  Positioning and layout is 100% managed by CinemachineDebug.  Client can pass in a Camera object when registering.  All client has to do is update the text, color, and visibility status of the widget.  Implementation details are hidden.